DDM Select Field

diegonvs opened the pull request.

jorgediaz-lr committed:
LPS-100538 portal-search: Execute searchEngine.initialize to force index creation in case it doesn't exist

https://issues.liferay.com/browse/LPS-100538

Commit:
LPS-110205 use request attribute instead, to avoid transactions clearing thread local caches

Commit:
LPS-110205 refactor

matuzalemsteles committed:
LPS-108474 Migrate FieldBase to React and add Adapter for compatibility

matuzalemsteles committed:
LPS-108474 Creates the connectStore to communicate with the Metal store on the React side

matuzalemsteles committed:
LPS-108474 Adds support for ReactAdapter as the frontline of Register

matuzalemsteles committed:
LPS-108474 Removes the ColorPicker component in Metal and adds the ReactColorPickerAdapter in the front line of the soy register

matuzalemsteles committed:
LPS-108474 SF

matuzalemsteles committed:
LPS-108474 Keeps FieldBase in Soy and React

We still need to keep implementing FieldBase in Soy and React. We can remove Soy when we have all fields in React.

This is because the FieldBase contentRenderer is a soy when the FieldBase is used in a Metal component and passing it to the React component requires that we do an `IncrementalDOM.patch` so that the element is rendered correctly in the tree, but that still has some challenges, maintaining this will be a bit dangerous so I’m keeping the two implementations together.

matuzalemsteles committed:
LPS-108474 stops giving error in the fieldInstance check

matuzalemsteles committed:
LPS-108466 Migrate Checkbox to React

matuzalemsteles committed:
LPS-108467 Migrate CheckboxMultiple to React

matuzalemsteles committed:
LPS-108474 Regen ColorPicker snapshots

matuzalemsteles committed:
LPS-108465 Migrate Captcha to React

matuzalemsteles committed:
LPS-108465 attach captcha events when window is loaded

matuzalemsteles committed:
LPS-108474 SF

matuzalemsteles committed:
LPS-108474 Regen ColorPicker snapshots

matuzalemsteles committed:
LPS-108465 Run scripts within Captcha markup

matuzalemsteles committed:
LPS-108465 Controls the refresh captcha on the React side

matuzalemsteles committed:
LPS-108465 Removes simplecaptcha implementation from the React side and adds events to simplecaptcha to attach events

matuzalemsteles committed:
LPS-108485 Migrate Separator to React

matuzalemsteles committed:
LPS-108465 SF

matuzalemsteles committed:
LPS-108481 Migrate Paragraph to React

matuzalemsteles committed:
LPS-108481 Regen snapshots

matuzalemsteles committed:
LPS-108481 stop giving error on the view screen

matuzalemsteles committed:
LPS-108482 Migrate Password to React

matuzalemsteles committed:
LPS-108482 Let React handle subsequent renderings

matuzalemsteles committed:
LPS-108481 Fixes tests

matuzalemsteles committed:
LPS-108483 Migrate Radio to React

matuzalemsteles committed:
LPS-108466 removes the use of refs on the checkbox adapter

matuzalemsteles committed:
LPS-108486 removes the use of refs in Validation

matuzalemsteles committed:
LPS-108486 Migrate Text to React

matuzalemsteles committed:
LPS-108486 Add the language values in the hidden input

matuzalemsteles committed:
LPS-108479 SF

matuzalemsteles committed:
LPS-108479 Migrate Numeric to React

matuzalemsteles committed:
LPS-108479 SF

matuzalemsteles committed:
LPS-108478 Migrate KeyValue to React

matuzalemsteles committed:
LPS-108480 Fix regression

matuzalemsteles committed:
LPS-108479 Fix name undefined

matuzalemsteles committed:
LPS-108480 Regen snapshots

matuzalemsteles committed:
LPS-108486 fixes Text tests

matuzalemsteles committed:
LPS-108479 fixes Numeric tests

matuzalemsteles committed:
LPS-108479 SF

diegonvs committed:
LPS-108477 Migrate Grid field to React using ReactComponentAdapter

diegonvs committed:
LPS-108477  When the field is readOnly, disable the radio inputs

diegonvs committed:
LPS-108477 Pass some required properties to FieldBaseProxy

alee8888 committed:
LPS-107598 Add Domain as a filter parameter to the display context

alee8888 committed:
LPS-107598 Add filter dropdown for Domain

alee8888 committed:
LPS-107598 Add accountEntryIds parameter and filter out Account Users for Company Users

alee8888 committed:
LPS-107598 Add conditional to filter by Account Users only

alee8888 committed:
LPS-107598 Add accountEntryIds attribute and use the search indexer

alee8888 committed:
LPS-107598 buildLang

matuzalemsteles committed:
LPS-108480 minor performance improvements and preventing memory leak in Adpater

matuzalemsteles committed:
LPS-108478 Some improvements and exports the KeyValue without the connectStore

matuzalemsteles committed:
LPS-108486 Checks the item to avoid errors when the value is undefined

diegonvs committed:
LPS-108477 Update and enhance coverage of DDM Grid field tests

matuzalemsteles committed:
LPS-108480 Migrate Options to React without DnD

The Drag and Drop implementation will be implemented in another commit.

victorg1991 committed:
LPS-109984 When elevating a child of a container take the container, not the parent of the container

veroglez committed:
LPS-109298 Adding new tabs fragment

veroglez committed:
LPS-109298 Adding new language key for tabs

veroglez committed:
LPS-109298 Updating language files

p2kmgcl committed:
LPS-109984 Remove elevate conditions

veroglez committed:
LPS-109407 Removing bold font and background when a column in the tree is selected

sandrodw3 committed:
LPS-110235 Added MappingSelector tests file with test about rendering correct selects in content pages

adolfopa committed:
LPS-108921 Use Asset Entry item selector to select a web content

sandrodw3 committed:
LPS-110235 Added tests about rendering correct selects in display pages

matuzalemsteles committed:
Revert "LPS-108480 Migrate Options to React without DnD"

This reverts commit 9c3821ff13046716e6f0b138f2fd2dcd924bf0a3.

matuzalemsteles committed:
LPS-108478 Fixes tests in the Options field

matuzalemsteles committed:
LPS-108474 Fixes spacing problems

p2kmgcl committed:
LPS-110262 Allow setting initialState in ControlsProvider

sandrodw3 committed:
LPS-110235 Added test for checking onMappingSelect call when mapping from a content

sandrodw3 committed:
LPS-110235 Added test for checking onMappingSelect call when mapping from a structure

sandrodw3 committed:
LPS-110235 Added test for checking onMappingSelect call when unmapping

p2kmgcl committed:
LPS-110262 Add missing label

sandrodw3 committed:
LPS-110235 Remove unnecessary call to loadFields

sandrodw3 committed:
LPS-110235 Don't use mappedItemTitle

ambrinchaudhary committed:
LPS-108921 Single selection mode

sandrodw3 committed:
LPS-110235 Unmap when selecting new item or changing source type

sandrodw3 committed:
LPS-110235 Set selected item field

matuzalemsteles committed:
LPS-108474 Fix regression of opening the sidebar

sandrodw3 committed:
LPS-110235 Set source type id when creating it

matuzalemsteles committed:
LPS-108474 Fixes validation of values ​​that change in the Adapter

matuzalemsteles committed:
LPS-108477 SF

ambrinchaudhary committed:
LPS-108921 Add the status in all the display types

ambrinchaudhary committed:
LPS-108921 SF

matuzalemsteles committed:
LPS-108474 Adds default values ​​for configuration to track subsequent changes

matuzalemsteles committed:
LPS-108479 Stop infinite looping when changing the value

matuzalemsteles committed:
LPS-108474 Fix error in Rule Editor that does not enable the save button

matuzalemsteles committed:
LPS-108474 Fixes error in reordering Options fields

Options is still in Metal+Soy, it updates the key when the reordering changes, but we don't change it on the adapter, because we were ignoring it, makes React understand that it must reuse the DOM reference but we want it to destroy and rebuild again.

carolmariaabb committed:
LPS-110338 Sync options of default language with options from other languages

p2kmgcl committed:
LPS-110262 Make remove button accesible

p2kmgcl committed:
LPS-110262 Missing language key

p2kmgcl committed:
LPS-110262 Add tests for PageStructureSidebar

sandrodw3 committed:
LPS-110340 Added test file for ItemSelector component

sandrodw3 committed:
LPS-110340 Added tests related to item title

sandrodw3 committed:
LPS-110340 Added missing button label

Commit:
LPS-108562 Update the flag of Slovakia

sandrodw3 committed:
LPS-110340 Added tests related to openInfoItemSelector function

veroglez committed:
LPS-110364 Adding test for FloatingToolbar component

sandrodw3 committed:
LPS-110364 Remove unnecessary test

matuzalemsteles committed:
LPS-108474 Makes fields in tabbed mode with keys and refs uniques

matuzalemsteles committed:
LPS-108467 Fixes doubling of values

diegonvs committed:
LPS-108484 Migrate Select DDM field to React

brandizzi committed:
LPS-100538 portal-search-elasticsearch7-impl: Realize the test case for Elasticsearch 6 and extractlogic from ES engine test to the fixture we need.

https://issues.liferay.com/browse/LPS-100538

brandizzi committed:
LPS-100538 portal-search-elasticsearch6-impl: Realize the test case for Elasticsearch 6 and extractlogic from ES engine test to the fixture we need.

https://issues.liferay.com/browse/LPS-100538

adolfopa committed:
LPS-108921 Let callers choose single or multiple select

Item Selector already has a 'singleSelect' attribute, that signals the
frontend component that the view will return one or multiple results.
Ideally, that same frontend component would send the parameter to the view,
but for the moment, we need to duplicate that info in the Js configuration
and the URL. We'll fix that in a different LPS.

We're keeping the 'multipleSelection' parameter of the Asset Browser
for compatibility reasons. Eventually, we'll remove all Asset Browser
usages and use Item Selector in its place.

adolfopa committed:
LPS-108921 Show status only when displaying non approved entries

Displaying it in the other cases is redundant, as all entries
will be in approved status.

adolfopa committed:
LPS-108921 Keep old single selection around for backwards compatibility

We'll remove all old usages after GA2. For the moment, keep things
working to keep backwards compatibility.

matuzalemsteles committed:
LPS-108474 Simplifies the unique value of the ref and key

matuzalemsteles committed:
LPS-108486 Compare if the received value is different from the previous one

matuzalemsteles committed:
LPS-108474 SF

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:
```diff
@@ -12,449 +12,337 @@
+
+										// Forces the active element to be blurred.
+										if (document.activeElement) {
+											document.activeElement.blur();
```
hey @diegonvs why do we have to force this here on click?

diegonvs replied:
I just left the same code as the original component here

matuzalemsteles replied:
Hmm OK. This is strange, @brunobasto do you know which case this covers? Just so we know what it covers and put a better description for future people.

brunobasto replied:
Humm.. I think it might be safe to remove. Doesn't make much sense.

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:
```diff
@@ -12,449 +12,337 @@
+
+								setCurrentValue(newValue);
+
+								event.preventDefault();
```
why don't we do this before starting the checks above?

diegonvs replied:
I kept the same order as the legacy logic.

brandizzi committed:
LPS-100538 portal-search: Instead of using utils, inject DS dependencies to allow testing.

https://issues.liferay.com/browse/LPS-100538

matuzalemsteles reviewed (commented):
@diegonvs I did a partial review, I still want to wait for https://github.com/brunobasto/liferay-portal/pull/1113, for him to come in and we can do some manual tests and also run the poshi tests.

victorg1991 committed:
LPS-110308 Check that elements are present

victorg1991 committed:
LPS-110308 Add tests for FragmentContent

matuzalemsteles committed:
LPS-108477 Fixes error when the field Grid is not initialized on the view screen

matuzalemsteles committed:
LPS-108477 Skip tests flaky

matuzalemsteles committed:
LPS-108465 SF

victorg1991 committed:
LPS-110308 Include languageId in the config request too

victorg1991 committed:
LPS-110308 Add tests for resolveEditableValues function

matuzalemsteles committed:
LPS-108474 Fixes error when changing the field type of the field

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:
```diff
@@ -12,449 +12,337 @@
+							data-testid={`dropdownItem-${index}`}
+							key={`dropdown-option-${index}`}
+							label={option.label}
+							onClick={event => {
```
@diegonvs this `onClick` logic is very big in JSX, we can move it above JSX to clean JSX and decrease noise in the code

diegonvs replied:
Splitting up! :)

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/components/HiddenSelectInput.es.js:
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+const HiddenSelectInput = ({multiple, name, options, readOnly, value}) => {
+	if (!readOnly) {
```
It is a little strange to have this conditional here, I think it might go to the Trigger component in this case. This component does not need to know this.

diegonvs replied:
Makes sense, this component can be only presentational. :)

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/components/HiddenSelectInput.es.js:
```diff
@@ -0,0 +1,58 @@
+		return (
+			<ClaySelect
+				aria-label="select"
+				className="form-control hide"
```
Because instead of hiding the element with the `hide` class, do not use the `hidden` attribute.

diegonvs replied:
✅

diegonvs replied:
Apparently `hidden` property doesn't apply a `.hide` class to the select

matuzalemsteles replied:
The idea is that we don't need a `hide` class, a hidden property won't show up visually.

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/components/VisibleSelectInput.es.js:
```diff
@@ -0,0 +1,127 @@
+						/>
+					) : (
+						value.map(currentValue => {
+							return options.map(option => {
```
Do we need two maps? I think this can be simplified just by going through the options and checking if `value.includes(option.value)` exists.

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/components/VisibleSelectInput.es.js:
```diff
@@ -0,0 +1,127 @@
+											key={`option-selected-${option.label}`}
+											label={option.label}
+										/>
+									);
```
Since it will only appear once, we can remove it from here and simplify the rule, we just let the map go through to assemble the labels when they are multiple.

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/components/VisibleSelectInput.es.js:
```diff
@@ -0,0 +1,127 @@
+					)}
+					disabled={readOnly}
+					id={id}
+				>
```
In summary of the comments above I think this could look like this:

```jsx
{(value.length === 0 || value.length === 1 && !multiple) ? (
	<OptionSelected
		isPlaceholder={value.length === 0 ? true : false}
		// This can go to a variable
		label={value.length === 0 ? triggerPlaceholder : options.find(option => option.value === value[0]).label}
	/>
) : value.map(item => {
	const option = options.find(option => option.value === item);
	return (
		<LabelOptionListItem
			key={`${option.value}-${option.label}`}
			onCloseButtonClicked={
				onCloseButtonClicked
			}
			option={option}
		/>
	);
})}
```

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:
```diff
@@ -12,449 +12,337 @@
+	const [expand, setExpand] = useState(false);
+
+	return (
+		<div>
```
Replace by `<></>` (Fragment)

diegonvs replied:
✅

istvansajtos committed:
LPS-109784 Enable auto login for /c/document_library/get_file/* path

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:
```diff
@@ -12,449 +12,337 @@
+				value={currentValue}
+				{...otherProps}
+			/>
+			<ClayDropDown.Menu
```
@diegonvs we will have to implement the focus manually here, you can check an example in Text

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:
```diff
@@ -12,449 +12,337 @@
+									}
+								}
+								else {
+									newValue = toArray(itemValue);
```
instead of `toArray` just one `[itemValue] `is enough.

diegonvs replied:
✅

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:
```diff
@@ -12,449 +12,337 @@
+				assertOptionParameters({multiple, option, valueArray})
+			)
+		)
+		.filter(({value}) => value !== '');
```
this could look like this:

```js
const newOptions = [
	...options.map((option, index) => ({
		...assertOptionParameters({multiple, option, valueArray}),
		separator:
			Array.isArray(fixedOptions) &&
			fixedOptions.length > 0 &&
			index === options.length - 1,
	})),
	...fixedOptions.map(option =>
		assertOptionParameters({multiple, option, valueArray})
	)
].filter(({value}) => value !== '');
```

diegonvs replied:
✅

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:
```diff
@@ -12,449 +12,337 @@
+ * Some parameters on each option
+ * needs to be prepared in case of
+ * multiple selected values(when the value state is an array).
+ *
```
remove line

matuzalemsteles commented on modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:
```diff
@@ -12,449 +12,337 @@
  * details.
  */
 
```
Remember to import the register.

```js
import './SelectRegister.soy';
```

matuzalemsteles reviewed (changes requested):
hey @diegonvs I think we can simplify some more logic here, i left some comments.

Commit:
LPS-109916 Create basic slug parser

Commit:
LPS-109916 Change URLs to make them readable to humans

Commit:
LPS-109916 SF

rodrigopaulino committed:
LPS-110205 Use request shared attributes

sandrodw3 committed:
LPS-110174 Stop double click event propagation

pavel-savinov committed:
LPS-110513 Check layout content model permission as well

pavel-savinov committed:
LPS-110513 Product menu side navigation can be null

veroglez committed:
LPS-109298 Improving accessibility and adapting the code for ie11

crodriguezy committed:
LPS-110589 Make document type fields localizable

sandrodw3 committed:
LPS-110174 Move visit and getName to separated functions

sandrodw3 committed:
LPS-110174 Wrap nodes calculation in useMemo

boton committed:
LPS-109916 Append slash only when is needed

gamerson committed:
LPS-110131 update project-template tokens

diegonvs committed:
LPS-108484 Split logic outside JSX

diegonvs committed:
LPS-108484 Makes HiddenSelectInput presentational

diegonvs committed:
LPS-108484 Use `hidden` property instead of using `.hide` class

Commit:
LPS-110669 Fix page builder column resizer button

balazssk committed:
LPS-110597 Adapting LPS-88378 for draft pages

jonathanmccann committed:
LPS-110606 Do not display the portlet in the case of Related Assets since the user cannot add new assets from this portlet

jorgediaz-lr committed:
LPS-108113 Don't export AssetDisplayPageEntry object the second time _exportAssetDisplayPage code is executed to avoid a loop that causes a stackoverflow

brandizzi committed:
LPS-100538 portal-search-test-util: Create an interface for a fixture to be implemented by ES modules, and a test case using this fixture.

https://issues.liferay.com/browse/LPS-100538

diegonvs committed:
LPS-108484 Imports SelectRegister on top of the React component

Currently is necessary importing the Soy register file on top of the component for preventing some rendering issues

diegonvs committed:
LPS-108484 Avoids wrong positioning on the Dropdown panel passing directly the Trigger element ref as prop instead of just setup the reference of the target of the click event, that can be a element inside the trigger

diegonvs committed:
LPS-108484 Make Dropdown Items select element when allowing multiple select items

diegonvs committed:
LPS-108484 Turns the field disabled when readOnly property is true

diegonvs committed:
LPS-108484 Use fragments instead of a empty div element

diegonvs committed:
LPS-108484 Unnecessary toArray call, since the itemValue at this point is definitely a string

diegonvs committed:
LPS-108484 Unnecessary preventDefault and stopPropagation usages

diegonvs committed:
LPS-108484 Source Formatting

diegonvs committed:
LPS-108484 Missing dependency

diegonvs committed:
LPS-108484 Refactoring normalize options function

diegonvs committed:
LPS-108484 Refactoring VisibleSelectInput component, using memoization

diegonvs committed:
LPS-108484 Unecessary blurring the focused element

rachaelkoestartyo committed:
LRAC-5502 proper warning for hasConfiguration check

rachaelkoestartyo committed:
LRAC-5502 check isActive

rachaelkoestartyo committed:
LRAC-5502 no longer throws exception

rachaelkoestartyo committed:
LRAC-5502 more isActive checks

Commit:
LPS-109916 Change the way to work with history.push to make possible handle the url and convert it to a friendlyUrl

rachaelkoestartyo committed:
LRAC-5502 set back to false if no configurations

diegonvs committed:
LPS-108484  Adds a divider for handling fixed  select options

diegonvs committed:
LPS-108484 Memoize normalization functions

Commit:
LPS-109916 Create own Link component which use react-router's Link to be able to handle the url

diegonvs committed:
LPS-108484 Splits DropdownItem to another component

Commit:
LPS-109916 SF

diegonvs committed:
LPS-108484 Adds tests for Select field

diegonvs committed:
LPS-108484 Adds navigation via keyboard to Select DDM field

shuyangzhou committed:
LPS-110705 Sync ES threadpools to avoid race condition

shuyangzhou committed:
LPS-110705 Remove all IdempotentRetryAssert usages outside search modules.

JoyceWang08211 committed:
LRQA-56862 Add test suite for site template

shuyangzhou committed:
LPS-110705 Remove unnecessary tests, no need to on purposely making troubles for ourselves.

Sharryxue committed:
LRQA-56504 Add additional version for upgrade to add sub-organization

JoyceWang08211 committed:
LRQA-56870 Creation of Upgrades Staging component

Jeremy-Ch committed:
LRQA-56837 Add path for the section background image

Jeremy-Ch committed:
LRQA-56837 Update tests to use the suitable path

shinnlok committed:
LRAC-5502 Fix logic

shinnlok committed:
LRAC-5502 Since these exceptions are unexpected, we want to pass along the stacktrace.

pavel-savinov committed:
LPS-110328 SF - Order

JoyceWang08211 committed:
LRQA-56881 Fix test suite for export-import

Jeremy-Ch committed:
LRQA-56843 Ensure the documents are fully uploaded

liulukiwi committed:
LRQA-56814 Stable cases by adding column

pavel-savinov committed:
LPS-110573 Create draft layout in approved state if draft doesn't exist

liulukiwi committed:
LRQA-55939 Use json to add widget in case

sandrodw3 committed:
LPS-110262 buildLang

Commit:
COMMERCE-2636 talend-definition - remove ineffective code - super class implementation is empty

Commit:
COMMERCE-2636 talend-definition - follow pattern we used for other liferay components

Commit:
COMMERCE-2636 talend-definition - resources - add properties

Commit:
COMMERCE-2636 talend-definition - TEST - update

Commit:
COMMERCE-2636 talend-definition - implement BatchOtput to be able to independently submit batch file to batch engine

Commit:
COMMERCE-2636 talend-runtime - update sink implmentation to support properties updates

Commit:
COMMERCE-2636 talend-runtime - fix bugs and fine tune the code

p2kmgcl committed:
LPS-109298 Use editable image until we have dropzones

pavel-savinov committed:
LPS-108412 Shopw pending label in case of rejected or pending layout

pavel-savinov committed:
LPS-108412 Don't change denied layout to pending

pavel-savinov committed:
LPS-108412 Don't allow to submit denied page for publication from the editor

AliciaGarciaGarcia committed:
LPS-110232 add test to role service

4lejandrito committed:
LPS-110232 Simplifying with DepotTestUtil

jkappler committed:
LPS-108921 Reorder

victorg1991 committed:
LPS-110664 Use correct action and adapt code to receive a list

victorg1991 committed:
LPS-110664 Add tests for the widget reducer

Commit:
LPS-110594 Add tests

Commit:
LPS-109916 Add lost useEffect

pavel-savinov committed:
LPS-110607 Regenerate service

matuzalemsteles committed:
LPS-108474 Fix rebase regression

petershin committed:
LPS-110693 Blind copy init-compile task

shuyangzhou committed:
LPS-110705 Sync run for test

inacionery committed:
LPS-108505 Fix configurationPid path

inacionery committed:
LPS-108505 Scope must be per system

inacionery committed:
LPS-108505 Modified must call deactivate

Commit:
LPS-110594 SF

bryceosterhaus committed:
LPS-110697 - Update clay-* dependencies

jkappler committed:
LPS-110695 Search for the themeId, since it can be different from the themeName

petershin committed:
LPS-110693 Check the release properties files

petershin committed:
LPS-110693 Check if a file has been deleted

petershin committed:
LPS-110693 Use up-to-date check for task avoidance

petershin committed:
LPS-110693 Rename directory

petershin committed:
LPS-110693 Simplify

pyoo47 committed:
LRCI-1124 Always mark slaves offlined by slave offline rules as PINNED

brianchandotcom committed:
LPS-108921 sort

brianchandotcom committed:
LPS-108921 Burrito

brianchandotcom committed:
LPS-110205 rename

brianchandotcom committed:
LPS-110573 as used

brianchandotcom committed:
LPS-110131 prep next

brianchandotcom committed:
LPS-110131 prep next

shuyangzhou committed:
LPS-110705 Restore unit test behavior

brianchandotcom committed:
LPS-107598 Sort

brianchandotcom committed:
LRCI-1124 prep next

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/analytics/.gitrepo'.

shuyangzhou committed:
LPS-110705 Strict search to prevent race condition

Commit:
LRQA-56953 Add some line brakes to AppBuilder#DeleteFormViewFieldFromObject 3

Commit:
LRQA-56958 Modify locator for background image of fragment

Commit:
LRQA-56958 Add parameter for reuseable

Commit:
LRQA-56959 Modify locator name

Commit:
LRQA-56959 Replace old locator by new one

Commit:
LRQA-56959 Modify case to make sure fragment entry published successfully

austinchiang committed:
LRQA-56965 Type in CKEditor

brianchandotcom committed:
LPS-110705 move to one method

brianchandotcom committed:
LPS-110705 SF

austinchiang committed:
LPS-110710 Update path for unit tests

austinchiang committed:
LPS-110711 Add test.properties for Redirect

Commit:
LRQA-56954 Modify cases to adapt new pagination

Commit:
LRQA-56954 Use JSONWS add multiple users to decrease running time

Commit:
LRQA-56954 Remove abandoned locator

brianchandotcom committed:
LPS-108394 prep next

brianchandotcom committed:
LPS-108394 prep next

liulukiwi committed:
LRQA-56979 Ignore this case for LRQA-55174

liulukiwi committed:
LRQA-56981 Ignore this case for LRQA-56815

liulukiwi committed:
LRQA-56983 Ignore this case for LRQA-56819

liulukiwi committed:
LRQA-56984 Ignore this case for LRQA-56915

liulukiwi committed:
LRQA-56985 Ignore this case for LRQA-56204

matuzalemsteles commented:
@diegonvs I will close this since it was sent to Timão, right?
